doc/spinel-protocol-src/spinel-prop-core.md

```diff
@@ -307,6 +308,31 @@ response from the NCP acknowledging the command (with `CMD_VALUE_IS`).
 Once that acknowledgement is received the host may enter the low-power
 state.
 
+### PROP 4104: PROP_UNSOL_UPDATE_FILTER {#prop-unsol-update-filter}
+
+* Required only if `CAP_UNSOL_UPDATE_FILTER` is set.
+* Type: Read-Write
+* Packed-Encoding: `A(I)`
+* Default value: Empty.
+
+Contains a list of properties which are excluded from generating
+unsolicited value updates. This property **MUST** be empty after reset.
+
+Hosts **SHOULD NOT** add properties to this list which are not
+present in `PROP_UNSOL_UPDATE_LIST`. If such properties are added,
+the NCP **MUST** ignore the unsupported properties.
```

Comment:
Was trying to avoid possible future interop problems by explicitly making sure that unknown properties are ignored. Since unknown properties will obviously not be generating unsolicited updates, it seems fairly harmless. An implementation may print out a warning to the debug stream.

Note that the error is still detectable: If you VALUE_SET unsupported properties, the resulting VALUE_IS would contain only the supported properties of that set(since the unsupported properties would be ignored). If an implementation cares that much about getting this right then it needs to make sure that it checks `PROP_UNSOL_UPDATE_LIST` first.